contracts/CO30_Andrew_HotInHerre.ruha/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,5 @@
+<font color='#E3D310'>PRIMARY</font>
+Eliminate all hostile enemies in the IDAP Center in Kaaranmännikkö.
```

Comment:
```suggestion
- Secure the IDAP Center in Kaaranmännikkö
```
Should be in jot note format